spec-main/api-browser-window-spec.ts

```diff
@@ -1120,12 +1122,12 @@ describe('BrowserWindow module', () => {
             transparent: true
           });
 
-          const maximize = emittedOnce(w, 'resize');
+          const maximize = emittedOnce(w, 'maximize');
```

Comment:
No, these were not copy-and-paste typos. Maximizing transparent windows on Windows is done by resizing the window rather than actually maximizing it. If you change it back, I believe it should pass the Windows test it is currently timing out on. If it is failing on Linux with the previous implementation, then there may just need to be a test for each system.

Reply:
Thanks for the context! If it's Windows-specific to transparent windows, then yea, perhaps this version should be a Windows-only test and the others can test with the regular `maximize` events, or simply a ternary there in the `emittedOnce` call which selects the event based on platform.